packages/nextjs/pages/index.tsx

```diff
@@ -141,17 +184,24 @@ const Home: NextPage = () => {
             <h2 className="mb-0 text-5xl font-bold">ABI Ninja</h2>
             <p className="">Interact with any contract on Ethereum</p>
             <div className="my-4">
-              <select
-                className="select select-sm w-36 max-w-xs bg-slate-50"
-                value={network}
-                onChange={e => setNetwork(e.target.value)}
-              >
-                {networks.map(network => (
-                  <option key={network.id} value={network.id}>
-                    {network.name}
-                  </option>
-                ))}
-              </select>
+              <Select
+                defaultValue={groupedOptions["Mainnets"].options[0]}
+                instanceId="network-select"
+                options={Object.values(groupedOptions)}
+                onChange={option => setNetwork(option ? option.value.toString() : "")}
+                components={{ Option: IconOption }}
+                isSearchable={!isMobile}
+                className="text-sm w-44 max-w-xs bg-white relative"
+                theme={theme => ({
+                  ...theme,
+                  colors: {
+                    ...theme.colors,
+                    primary25: "#efeaff",
+                    primary50: "#c1aeff",
+                    primary: "#551d98",
+                  },
```

Comment:
Manually overriding default colors because it felt much easier than passing `unstyled` prop and manually handle all styling. Could reference somehow the colors from tailwind configuration file instead of hardcoding the hex values?

Reply:
Yup yup, also it's fine if we hardcode colors coz there is no proper way to get tailwind configured colors from the config 🙌